autogen/agentchat/contrib/qdrant_retrieve_user_proxy_agent.py

```diff
@@ -255,11 +255,11 @@ def create_qdrant_from_dir(
 
 def query_qdrant(
     query_texts: List[str],
-    n_results: int = 10,
-    client: QdrantClient = None,
-    collection_name: str = "all-my-documents",
-    search_string: str = "",
-    embedding_model: str = "BAAI/bge-small-en-v1.5",
+    n_results: Optional[int] = 10,
+    client: Optional[QdrantClient] = None,
+    collection_name: Optional[str] = "all-my-documents",
+    search_string: Optional[str] = "",
+    embedding_model: Optional[str] = "BAAI/bge-small-en-v1.5",
     qdrant_client_options: Optional[Dict] = {},
 ) -> List[List[QueryResponse]]:
```

Comment:
yeah, it does match the existing datatype, previously it was `Dict`, now its updated to the `List[List[QueryResponse]]`

@thinkall